app/services/scrape_matches_service.rb

```diff
@@ -0,0 +1,70 @@
+# TODO: We'll need to install the drivers to work on Heroku
+require 'nokogiri'
+
+class ScrapeMatchesService
+  attr_reader :urls, :browser
+
+  def initialize
+    # ids pulled from the UEFA website to scrape matches
+    @urls = []
+    group_stages_ids = [33673, 33674, 33675]
+    group_stage_url = 'https://www.uefa.com/uefaeuro-2020/fixtures-results/#/md/'
+    group_stages_ids.each { |id| @urls << "#{group_stage_url}#{id}" }
```

Comment:
I'm probably being too picky here, but shouldn't it be a `map`? :)

Reply:
lol i used `.each` because i was adding urls into the array on two separate occasions (group_stage and knockout). i thought it would be confusing if one was a `map` and the other was an `each`

Reply:
Haha yeah I thought there must be a reason behind it 😆